protractor-docker/run-tests.sh

```diff
@@ -0,0 +1,12 @@
+docker -v
```

Comment:
Let's rename the file to "docker-setup.sh" and give it "chmod +x" permissions. See comments above.